src/lib/updateVersion.js

```diff
@@ -0,0 +1,23 @@
+const fs = require("fs");
+const path = require("path");
+const getVersion = require("./getVersion.js");
+const chalk = require("chalk");
+
+module.exports = function updateLastVersionUsed() {
+  // Check version in .gluestick file
+  const gluestickDotFile = path.join(process.cwd(), ".gluestick");
+  var fileContents = fs.readFileSync(gluestickDotFile, {encoding: "utf8"})
+  // We won't know how old this version is, it might have still have the 'DO NOT MODIFY' header
+  fileContents = fileContents.replace("DO NOT MODIFY", "");
```

Comment:
This is just a safety check to strip it because we don't know how old the dot file is, and it might have one. We don't add it back in